strings/ctype-czech.c

```diff
@@ -336,24 +311,23 @@ my_strnxfrm_czech(CHARSET_INFO *cs __attribute__((unused)),
 
 
 /*
-	Neform�ln� popis algoritmu:
-
-	proch�z�me �et�zec zleva doprava
-	konec �et�zce pozn�me podle *p == 0
-	pokud jsme do�li na konec �et�zce p�i pr�chodu 0, nejdeme na
-		za��tek, ale na ulo�enou pozici, proto�e prvn� a druh�
-		pr�chod b�� sou�asn�
-	konec vstupu (pr�chodu) ozna��me na v�stupu hodnotou 1
-
-	na�teme hodnotu z t��d�c� tabulky
-	jde-li o hodnotu ignorovat (0), sko��me na dal�� pr�chod
-	jde-li o hodnotu konec slova (2) a je to pr�chod 0 nebo 1,
-		p�esko��me v�echny dal�� 0 -- 2 a prohod�me
-		pr�chody
-	jde-li o kompozitn� znak (255), otestujeme, zda n�sleduje
-		spr�vn� do dvojice, dohled�me spr�vnou hodnotu
-
-	na konci p�ipoj�me znak 0
+  Informal description of the algorithm:
+
+  we pass the chain from left to right
+  we know the end of the string by *p == 0
+  if we reached the end of the string on transition 0, then we don't go to
+    start, but to the saved position, because the first and second
+    the passage runs concurrently
+  we mark the end of the input (transition) with the value 1 on the output
+
+  then we load the value from the sorting table
+  if the value is ignore (0), we jump to the next pass
+  if the value is the end of the word (2) and it is a 0 or 1 transition,
+    we skip all the other 0 -- 2 and switch transitions
+  if it is a composite character (255), we test whether it follows
+    correct to the pair, we find the correct value
+
+  then we add the character 0 at the end
```

Comment:
Tabs with this block too please, consistent with the rest of the comment blocks.